contracts/test/forkMainnet/AMMWrapper/Setup.t.sol

```diff
@@ -11,22 +11,20 @@ import { getEIP712Hash } from "contracts-test/utils/Sig.sol";
 
 contract TestAMMWrapper is StrategySharedSetup {
     uint256 constant BPS_MAX = 10000;
+    bytes32 public constant relayerValidStorageId = 0x2c97779b4deaf24e9d46e02ec2699240a957d92782b51165b93878b09dd66f61; // keccak256("relayerValid")
 
     uint256 userPrivateKey = uint256(1);
     uint256 otherPrivateKey = uint256(2);
 
     address user = vm.addr(userPrivateKey);
-    address feeCollector = address(0x133701);
-    address relayer = address(0x133702);
+    address owner = makeAddr("owner");
+    address feeCollector = makeAddr("feeCollector");
+    address relayer = makeAddr("relayer");
+    address psOperator;
```

Comment:
This can also be generated by `makeAddr`?
```suggestion
    address psOperator = makeAddr("psOperator");
```

Reply:
Adjusted in 6a94edd8d0af285b2e3c4c4aa284e5dcbe77c4b1